src/client/features/enrichment/enrichment-menu.js

```diff
@@ -4,7 +4,37 @@ const { Tab, Tabs, TabList, TabPanel } = require('react-tabs');
 
 class EnrichmentMenu extends React.Component {
 
+  /**
+   *  @description Hides nodes based on adjusted p_value, determined by on-screen slider
+   */
+  sliderUpdate(){
+    const cy = this.props.cySrv.get();
+
+    //get value from slider
+    let sliderVal = document.getElementById('enrichment-p_value-slider').value;
+
+    //compare p_values and hide if outside of chosen threshold
+    cy.nodes().forEach(node => {
+      if(node.data('p_value') > sliderVal)
+        node.addClass('hidden');
+      else
+        node.removeClass('hidden');
+    });
+  }
+
   render(){
+
+    const slider = [
+      //set min = 0.0001 to prevent all nodes from being hidden which will occur if min = 0
```

Comment:
OK I see you've fixed this min to 1E-03. That could get weird if all the nodes are < 1E-08. That kind of adjusted p-value is totally meaningless, but I digress.

Is it weird to fix the range based upon the data? or is that a low priority 'enhancement' we can punt to the next issue/milestone?

Reply:
I thought about dynamic scaling, but I feel that it's for the next issue/milestone - implementation will get messy because the slider feature uses enrichment.css for the legend background, enrichment-stylesheet.js for the node color, and the .js files in the enrichment feature folder to render the slider

Reply:
OK leave for another day.